tests/conftest.py

```diff
@@ -59,3 +59,11 @@ def scanner(rule: Rule, config: Config, rule_path: str) -> Scanner:
     scanner = Scanner(config, rule_path)
     scanner.rules = [rule]
     return scanner
+
+
+@pytest.fixture
+def scanner_without_filters(rule: Rule, config: Config, rule_path: str):
```

Comment:
Some tests do use randomly seeded credentials, so they have a small change to be filtered out by a pattern filter (for example by randomly generating `aaaa` sub-string
This `scanner_without_filters` fixture allows to run regex related tests without filters that can remove some random creds

All other tests still using old `scanner` fixture

Reply:
UnitTests should avoid any "chance of".